lib/jazzy/sourcekitten/Rakefile

```diff
@@ -5,10 +5,10 @@ task :prepare do
   Dir.glob('*.framework').each do |fw|
     p fw
     Dir.chdir(fw) do
-      basename = File.basename(fw, '*.framework')
-      version = Dir.glob('Versions/*').sort.last
```

Comment:
Ah, in that case, `Dir.glob('Versions/*').map { |d| File.basename(d) }.sort.last` ?